### PR TITLE
fix: mode

### PR DESCRIPTION
**Pull request by eniola-ajala**
Close #23325

## Related Issue

Close #23325

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Comment by eniola-ajala** [comment]
Please review

**Comment by eniola-ajala** [comment]
close #23325

File: ivy/functional/frontends/paddle/manipulation.py [hunk 1]
```diff
@@ -173,3 +173,24 @@ def unstack(x, axis=0, name=None):
 
 
 absolute = abs
+
+#crrop
+# Load an image (you'll need to have an image file)
+image_path = "input_image.jpg"  # Replace with your image file path
```
Comment (umairjavaid) [inline ivy/functional/frontends/paddle/manipulation.py:179]: Please remove this code. This is not relevant.

**Comment by umairjavaid** [comment]
Can you kindly link the issue and the paddle doc link for the function that you are trying to implement. I can not find it

File: mode [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import paddle
```
Comment (umairjavaid) [inline mode:1]: This is not how you how implement a function. Can you kindly share the todo list

**Review by umairjavaid — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/paddle/manipulation.py (hunk 1 above)
Comment (eniola-ajala) [reply ivy/functional/frontends/paddle/manipulation.py:179]: I have been trying to link the issue and the paddle doc bit i haven't been able to. 
Can you guide me through it, please?

Comment (eniola-ajala) [reply ivy/functional/frontends/paddle/manipulation.py:179]: Sure, I will remove the codes

**Review by umairjavaid — CHANGES_REQUESTED** [review]
You have to implement `paddle.mode(x, axis=- 1, keepdim=False, name=None)` function in the paddle frontend. Which will call the ivy.mode function. But before that you have to make sure the ivy.mode function is implemented in the ivy api. Likewise, you have to also make sure the mode function is implemented in all of the backends. If it is not, you can work on implementing that. Check https://unify.ai/docs/ivy/overview/contributing/open_tasks.html#ivy-experimental-api

File: ivy/functional/frontends/paddle/manipulation.py (hunk 1 above)
Comment (umairjavaid) [reply ivy/functional/frontends/paddle/manipulation.py]: You have to implement the frontend as explained here https://unify.ai/docs/ivy/overview/contributing/open_tasks.html#frontend-apis

**Comment by eniola-ajala** [comment]
I have been unable to link the issue to the development, can you help me with it?

File: ivy/functional/frontends/paddle/search.py [hunk 3]
```diff
@@ -87,6 +87,14 @@ def sort(x, /, *, axis=-1, descending=False, name=None):
 def topk(x, k, axis=None, largest=True, sorted=True, name=None):
     return ivy.top_k(x, k, axis=axis, largest=largest, sorted=sorted)
 
+#mode
+@with_supported_dtypes(
```
Comment (umairjavaid) [inline ivy/functional/frontends/paddle/search.py:91]: You also have to implement its test function

**Review by umairjavaid — CHANGES_REQUESTED** [review]

**Comment by MahmoudAshraf97** [comment]
Hello, are you still working on this?

**Pull request closed without merging**